web/html/src/manager/systems/all-list.tsx

```diff
@@ -68,6 +69,7 @@ export function AllSystems(props: Props) {
         searchField={<SearchField options={allListOptions} name="criteria" />}
         defaultSearchField={props.queryColumn || "server_name"}
         initialSearch={props.query}
+        initialItemsPerPage={props.pageSize}
```

Comment:
Here and elsewhere, setting this prop on a `Table` should not be needed, `TableDataHandler` already uses `window.userPrefPageSize` as the default if it's set.

Reply:
Oh OK! IIRC Luis asked me a long time ago to start getting away from  that global `window` variable... does it mean I can just forget about that now?

Reply:
Not really, we still need to pass the value from the server to the frontend somehow. Wherever it's available on `window` it's used automatically by `TableDataHandler` though so you don't need to manually pass it around after it's set.

Reply:
OK that means I probably only need the Java part of this PR

Reply:
done